### DISCUSSION TITLE
Archer C7 v1.1

### DISCUSSION THREAD

**Discussion by MrStark-0** [xchwarze/wifi-pineapple-cloner#34] [Q&A] [answered]
I see the Archer C7 v2 etc...

Is the Archer C7 v1.1 able to be used and flashed etc.

There is an [openWRT 19.07.7 release avail for  the Archer C7 v1](https://firmware-selector.openwrt.org/?version=19.07.7&target=ath79%2Fgeneric&id=tplink_archer-c7-v1 )

I don't see it listed anywhere on this repo as being compatible.

TIA

**Comment by xchwarze** [comment, marked as answer]
as far as I know, this router does not have enough ROM.
https://deviwiki.com/wiki/TP-LINK_Archer_C7_v1.x

**Reply by MrStark-0** [reply]
...and here I thought I got a good deal on this unit.

Specs showing 8 MB flash compared to the rest with 16 MB

https://openwrt.org/toh/tp-link/archer_c7

In case anyone else finds this.
